packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -74,22 +125,43 @@ const RelationInput = ({
               onMenuOpen={onSearchOpen}
               onMenuScrollToBottom={onSearchNextPage}
               placeholder={placeholder}
+              name={name}
             />
           </>
         }
         loadMore={
           !disabled &&
           labelLoadMore && (
-            <TextButton onClick={() => onRelationLoadMore()} startIcon={<Refresh />}>
+            <TextButton
+              disabled={relations.isLoading}
+              onClick={() => onRelationLoadMore()}
+              startIcon={
+                relations.isLoading ? (
+                  // To replace with loading prop on TextButton after DS release
+                  <LoaderWrapper>
+                    <Loader />
+                  </LoaderWrapper>
+                ) : (
+                  <Refresh />
+                )
+              }
+            >
               {labelLoadMore}
             </TextButton>
           )
         }
       >
-        <RelationList height={listHeight}>
-          {relations.isSuccess &&
-            relations.data.pages.flat().map((relation) => {
-              const { publicationState, href, mainField, id } = relation;
+        <RelationList overflow={totalNumberOfRelations > numberOfRelationsToDisplay && overflow}>
```

Comment:
Right now the markup is broken/ invalid. I'm not sure if we are able to keep the ol/ li structure with react-window or if we'd have to go and use div's instead with roles?

<img width="618" alt="Screenshot 2022-09-06 at 13 34 39" src="https://user-images.githubusercontent.com/2244375/188625744-f9b8588e-0740-40b7-b249-e402cfd53d28.png">